pkg/scheduler/core/generic_scheduler.go

```diff
@@ -162,7 +163,6 @@ func (g *genericScheduler) assignReplicas(
 			for i, cluster := range clusters {
 				targetClusters[i] = workv1alpha2.TargetCluster{Name: cluster.Name, Replicas: object.Replicas}
```

Comment:
I guess here will panic if matches `case policyv1alpha1.ReplicaSchedulingTypeDuplicated`, since the `targetClusters` hasn't been initialized.

Reply:
yes, i'll  fix it